frontend/src/utils/TaskUtils.ts

```diff
@@ -288,15 +236,29 @@ export const calcWeightedTaskPriority = (task: Task, roadmap: Roadmap) => {
   return weightedValue / avgWorkRating;
 };
 
-export const isRatedByUser = (user: RoadmapUser | UserInfo) => (
-  rating: Taskrating,
-) => rating.createdByUser === user.id;
+export const awaitsUserRatings = (
+  user: UserInfo,
+  roadmap: number | Roadmap,
+) => {
+  const roadmapId = typeof roadmap === 'number' ? roadmap : roadmap.id;
+  const type = getType(user.roles, roadmapId);
+  if (type === RoleType.Admin || type === RoleType.Business) {
+    const reps = user.representativeFor?.filter(
```

Comment:
Clients or customers would describe the value better
```suggestion
    const clients = user.representativeFor?.filter(
```